src/util.jl

```diff
@@ -28,28 +28,28 @@ end
 
 av_pointer_to_field(s::Array, name::Symbol) = av_pointer_to_field(pointer(s), name)
 
-function open_stdout_stderr(cmd::Cmd)
-    out = Base.PipeEndpoint()
-    err = Base.PipeEndpoint()
-    cmd_out = Base.PipeEndpoint()
-    cmd_err = Base.PipeEndpoint()
-    Base.link_pipe(out, true, cmd_out, false)
-    Base.link_pipe(err, true, cmd_err, false)
-
-    r = spawn(ignorestatus(cmd), (devnull, cmd_out, cmd_err))
-
-    Base.close_pipe_sync(cmd_out)
-    Base.close_pipe_sync(cmd_err)
-
-    # NOTE: these are not necessary on v0.4 (although they don't seem
-    #       to hurt). Remove when we drop support for v0.3.
-    Base.start_reading(out)
-    Base.start_reading(err)
-
-    return (out, err, r)
-end
-
-function readall_stdout_stderr(cmd::Cmd)
-    (out, err, proc) = open_stdout_stderr(cmd)
-    return (read(out, String), read(err, String))
-end
+#function open_stdout_stderr(cmd::Cmd)
+#    out = Base.PipeEndpoint()
+#    err = Base.PipeEndpoint()
+#    cmd_out = Base.PipeEndpoint()
+#    cmd_err = Base.PipeEndpoint()
+#    Base.link_pipe!(out, true, cmd_out, false)
+#    Base.link_pipe!(err, true, cmd_err, false)
+#
+#    r = run(pipeline(cmd, stdin=devnull, stdout=cmd_out, stderr=cmd_err); wait=false)
+#
+#    Base.close_pipe_sync(cmd_out)
+#    Base.close_pipe_sync(cmd_err)
+#
+#    # NOTE: these are not necessary on v0.4 (although they don't seem
+#    #       to hurt). Remove when we drop support for v0.3.
+#    Base.start_reading(out)
+#    Base.start_reading(err)
+#
+#    return (out, err, r)
+#end
+#
+#function readall_stdout_stderr(cmd::Cmd)
+#    (out, err, proc) = open_stdout_stderr(cmd)
+#    return (read(out, String), read(err, String))
+#end
```

Comment:
Let's just delete these lines.  (No reason to run CI again.)

Reply:
Indeed. I've removed them, but actually don't see options to cancel/edit CI jobs on appveyor, even though I'm logged in with access granted to the VideoIO org. Anyhow, would you mind cancelling the new jobs for the moment? thanks